index.html

```diff
@@ -305,23 +305,20 @@ <h3>
           "RESOURCE-TIMING-2#dom-performanceresourcetiming">PerformanceResourceTiming</dfn></code>
           interface:
         </p>
-        <ul data-dfn-for="PerformanceResourceTiming">
-          <li>The <dfn>initiatorType</dfn> attribute MUST return the
-          {{DOMString}} "<code>navigation</code>".
+        <ul>
+          <li>The `workerStart` getter steps are to perform the following steps:
+            <ol>
+              <li>Let |workerTiming| be |this|'s [=PerformanceNavigationTiming/service worker timing=].
+              <li>If |workerTiming| is null, then return |this|'s prototype's `workerStart`.
+              <li>Return |workerTiming|'s [=service worker timing info/start time=].
+            </ol>
           </li>
-          <li>The <dfn id='dom-PerformanceNavigationTiming-workerStart'><code>
-            workerStart</code></dfn> attribute MUST return the time immediately
-            before the user agent <a data-cite=
-            "service-workers#run-service-worker">ran the worker</a> (if the
-            <a>current document</a> has an <a data-cite=
-            "service-workers#dfn-containing-service-worker-registration">active
-            service worker registration</a> [[SERVICE-WORKERS]]) required to
-            service the request, or if the worker was already available, the
-            time immediately before the user agent <a data-cite=
-            "service-workers#fetchevent">fired an event named `fetch`</a> at
-            the <a data-cite="service-workers#dfn-active-worker">active
-            worker</a>. Otherwise, if there is no active worker this attribute
-            MUST return zero.
+          <li>The `fetchStart` getter steps are to perform the following steps:
+            <ol>
+              <li>Let |workerTiming| be |this|'s [=PerformanceNavigationTiming/service worker timing=].
+              <li>If |workerTiming| is null, then return |this|'s prototype's `fetchStart`.
```

Comment:
This changes the specified semantics. Does this match implementations? Are there tests to verify this?

Reply:
How does this change the semantics? This is what current implementations do.

Reply:
It changes it from the previously undefined behavior, by overriding `fetchStart` with the fetch event start time.
It's great that this matches implementations, but we'd need to add a test that verifies that. 
[Current tests](https://source.chromium.org/chromium/chromium/src/+/main:third_party/blink/web_tests/external/wpt/service-workers/service-worker/navigation-timing.https.html?q=workerStart%20f:wpt&ss=chromium) cover the order, but would be good to add a test that verifies that the values actually match. I filed https://github.com/w3c/navigation-timing/issues/148 to do that as a followup.